.github/workflows/trivy-pr-check.yaml

```diff
@@ -0,0 +1,18 @@
+name: 'Trivy Scan'
+on:
+  pull_request:
+jobs:
+  build:
+    name: Build
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-PIPELINE-0022   
**Check Name:** Ensure pipeline steps produce an SBOM   
**Severity:** HIGH   
**Message:** Consider adding SBOM generation tool in build job 'Build'  
[This comment was created by Aqua Pipeline]